system-tests/services/test-apps.js

```diff
@@ -638,9 +888,52 @@ describe("Services", function() {
         .getTableRowThatContains(serviceName)
         .contains("Running", { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
         .should("exist");
+
+      // Now click on the name
+      cy
+        .get(".page-body-content table")
+        .getTableRowThatContains(serviceName)
+        .get("a.table-cell-link-primary")
+        .contains(`${serviceName}`)
+        .click();
+
+      // open edit screen
+      cy
+        .get(".page-header-actions .dropdown")
+        .click()
+        .get(".dropdown-menu-items")
+        .contains("Edit")
+        .click();
+
+      // check if values are as expected
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .should("have.value", `/${Cypress.env("TEST_UUID")}/${serviceName}`);
+
+      cy
+        .root()
+        .getFormGroupInputFor("Memory (MiB) *")
+        .should("have.value", "10");
+      cy.root().getFormGroupInputFor("Command").should("have.value", cmdline);
```

Comment:
Let's try keep consistent throughout files and break a line in every chaining, it makes a easier to read and catch any human error.

```
cy.get('blabla')
    .foo()
    .bar()
```